src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue

```diff
@@ -26,13 +26,18 @@
 		class="button primary new-event"
 		@click="newEvent"
 		@shortkey="newEvent">
-		{{ $t('calendar', '+ New event') }}
+		<Plus :size="20" decorative />
```

Comment:
The `decorative` prop [has been removed](https://github.com/robcresswell/vue-material-design-icons/releases/tag/5.0.0) in vue-material-design-icons 5.0.0.